project/src/main/java/com/example/sk/javabootcamp/skjavabootcamp/exceptions/CartNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.example.sk.javabootcamp.skjavabootcamp.exceptions;
+
+public class CartNotFoundException extends RuntimeException {
```

Comment:
I wouldn't inherit from RuntimeException these should be just exceptions, RuntimeException is unchecked and I would argue these NotFound exceptions should be checked.